lib/talkable.rb

```diff
@@ -12,6 +12,7 @@ module Talkable
 
   class << self
     def configure(config = nil)
+      configuration.clean
```

Comment:
now you wouldn't be able to use `configure` several times for example in different places

Reply:
a developer has explicitly call `clean` and `configure` in order to get completely new configuration

Reply:
If I delete this line to clean will need:
```ruby
Talkable.configuration.clean
Talkable.configure(...)
```
I don't know if it's a worthy decision. Maybe will be better if I add a new method to Talkable and it's will:
```ruby 
Talkable.clean_configuration
Talkable.configure(...)
```
?